### PR TITLE
[Documentation]Add text on contributing-how-to-contribute.md

### PR DESCRIPTION
**Pull request by matheusmonte**
Signed-off-by: Matheus Clark dos Santos Monte <matheus.m@samsung.com>

### CODE REVIEW

File: docs/contributing-how-to-contribute.md [hunk 1]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
 ---
 
 ## How to Contribute
+
+Contribute is always welcome on our project, so if you`re interesting on contribute with us: THANKS!
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: break one line here

File: docs/contributing-how-to-contribute.md [hunk 2]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+
+Aiming at a healthy community and with an active voice of all its members, it will not be allowed:
+
++ Discrimination for any reason
++ Use of slang words
++ Threats
++ Bulling
++ Personal Offenses
+
+Together, we can maintain the health of our project, some responsibilities of the members:
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `we can maintain the health of our project. Here is some...`

File: docs/contributing-how-to-contribute.md [hunk 3]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+
++ To be educated
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `Be nice`

File: docs/contributing-how-to-contribute.md [hunk 4]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
++ Respond when mentioned
++ Help the beginners (remember, you were a beginner).
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `Support the beginners.`

File: docs/contributing-how-to-contribute.md [hunk 5]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+5. Is this bug something you have run into? Would you appreciate it being looked into faster?
+
+## Getting in Touch
+
+We don`t have a email list at this time, All our discursions are on github, please, start our repository or
+watch to be up-to-date with the project.
+
+## Coding Style
+
+Our project contains a lint, please, check this before procced with your PR.
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: Maybe it's worth to mention Prettier here.

File: docs/contributing-how-to-contribute.md [hunk 6]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+
+## Documentation
+
+If you have some doubt about the project, you can check the wiki or check the documentation on our [github](https://github.com/raphamorim/react-ape/tree/master/docs)
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `If you have some question or suggestion, open an issue. Let's chat about it!`

File: docs/contributing-how-to-contribute.md [hunk 7]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+
+## Documentation
+
+If you have some doubt about the project, you can check the wiki or check the documentation on our [github](https://github.com/raphamorim/react-ape/tree/master/docs) 
+
+Help us with documentation here.
+
+## License
+
+We are under [MIT license](https://opensource.org/licenses/MIT)
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `[MIT License](https://github.com/raphamorim/react-ape/blob/master/README.md)~

File: docs/contributing-how-to-contribute.md [hunk 8]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ ## Branch Organization
+
+ We keep our master in good shape, with tests passing (Almost all times :) ). Always use your up-to-date
+ master to based your contributings. 
+
+ If you want to use react-ape on production project, We suggest you use a tagged commit version on Master.
+
+ Some branchs to future features can be create, don`t use if you are not envolved on this feature development.
+
+ If you have some doubt about branchs to use, please send e-mail to mainteners or open a issue with your question.
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `please open a issue with your question.`

File: docs/contributing-how-to-contribute.md [hunk 9]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ + Fork our repository
+ + Code in your forked repository
+ + Run the lint in your final code
+ + Submit a pull request to marge or release branch (Please, check where is the best place in our roadmap).
+ + Wait for review, if you PR "need changes", change and commit at the same PR
+ + If everything is ok, you PR will be accept by our team. 
+
+ ## Branch Organization
+
+ We keep our master in good shape, with tests passing (Almost all times :) ). Always use your up-to-date
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: please remove this paragraph

File: docs/contributing-how-to-contribute.md [hunk 10]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ This repository is managed by Raphael Amorim, as maintener and creator of React-Ape, he( or another person indicate by him) 
+ will be responsible to approve new featuers, merges into master and review Pull requests.
+
+ Our project is hosted and managed at Github, all official release are linked on our offical repository following our 
+ branch organization.
+
+ If you want to submit some code for our repository, please, follow this steps:
+
+ + Fork our repository
+ + Code in your forked repository
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `+ Write your code`

File: docs/contributing-how-to-contribute.md [hunk 11]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ + Run the lint in your final code
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `+ Run lint and test task`

File: docs/contributing-how-to-contribute.md [hunk 12]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ + Submit a pull request to marge or release branch (Please, check where is the best place in our roadmap).
+ + Wait for review, if you PR "need changes", change and commit at the same PR
+ + If everything is ok, you PR will be accept by our team.
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `+ Done!`

**Comment by matheusmonte** [comment]
@raphamorim I apply your changes :)

File: docs/contributing-how-to-contribute.md [hunk 13]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
 ---
 
 ## How to Contribute
+
+Contribute is always welcome on our project, 
+so if you`re interesting on contribute with us: THANKS!
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `you're`

File: docs/contributing-how-to-contribute.md [hunk 14]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+so if you`re interesting on contribute with us: THANKS! 
+
+Below we will describe how you can be part of our community. 
+
+## Code of Conduct
+
+Aiming at a healthy community and with an active voice of all its members, it will not be allowed:
+
++ Discrimination for any reason
++ Use of slang words
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: remove this line please

File: docs/contributing-how-to-contribute.md [hunk 15]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+We can maintain the health of our project. Here is some responsibilities of the members:
+
++ First: BE NICE!
++ Respond when mentioned
++ Support the beginners (remember, you were a beginner).
++ Follow our codes guidelines in PR discourses and request for featues
+
+
+## Adding new features
+ Please check our [Roadmap](https://github.com/raphamorim/react-ape#roadmap), if you don`t find your feature,
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `don't`

File: docs/contributing-how-to-contribute.md [hunk 16]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
++ Respond when mentioned
++ Support the beginners (remember, you were a beginner).
++ Follow our codes guidelines in PR discourses and request for featues
+
+
+## Adding new features
+ Please check our [Roadmap](https://github.com/raphamorim/react-ape#roadmap), if you don`t find your feature, 
+ please use Issue on github projects to start a new discursion. 
+ 
+ If this receive a Go ahead, you can submit your patch as PR to the repository or one of our maintainers will
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `a "Go ahead"`

File: docs/contributing-how-to-contribute.md [hunk 17]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+ + Submit a pull request to marge or release branch (Please, check where is the best place in our roadmap).
+ + Wait for review, if you PR "need changes", change and commit at the same PR
+ + If everything is ok, you PR will be accept by our team. 
+ + Done!
+
+ ## Branch Organization
+
+ If you want to use react-ape on production project, We suggest you use a tagged commit version on Master.
+
+ Some branchs to future features can be create, don`t use if you are not envolved on this feature development.
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: `don't`

File: docs/contributing-how-to-contribute.md [hunk 18]
```diff
@@ -5,3 +5,89 @@ sidebar_label: How to Contribute
...
+
+ If you have questions about project, please open a issue with your question.
+
+## Bug tracking
+
+You can help us diagnose and fix existing bugs by asking and providing answers for the following:
+
+1. Is the bug reproducible as explained?
+2. Is it caused by environments (for instance, on different browsers or devices)?
```
Comment (raphamorim) [inline docs/contributing-how-to-contribute.md]: nit: `It's`

**Comment by matheusmonte** [comment]
Done, please, check :)

**Pull request merged**